### DISCUSSION TITLE
Reboot M1 remotely?

### DISCUSSION THREAD

**Discussion by distancesprinter** [GerryDazoo/Slinger#328] [General]
Is there a way (even if it is not a feature built into this software) to remotely reboot a Slingbox?  Sometimes my audio sounds fuzzy, but after power cycling, it clears up.  I'd love to do it remotely.

**Comment by blasco403** [comment]
I use a cheap Wi-Fi based power plug I can get to from anywhere for slingbox's as well as the STB attached to it.

They use a cloud based service for access much like the sling servers were providing (locating and connecting).

I have an IoT network configured that I connect all the Wi-Fi power plugs to. This keeps them isolated from my internal networks.

**Reply by pfuller88** [reply]
yes, the wifi power bar/plug works great.  i use this one, there are lots of choices.

Kasa Smart Plug Power Strip KP303, Surge Protector with 3 Individually Controlled Smart Outlets and 2 USB Ports, Works with Alexa & Google Home, No Hub Required , White

**Reply by mdehrlich** [reply]
I have my slingbox and my cable box (FIOS) on a wifi plug.  That way I reboot both if anything goes south.  I have my slinger PI on another plug, but I am thinking of putting them all together?
